src/controllers/midi/midicontroller.cpp

```diff
@@ -13,6 +13,16 @@
 #include "moc_midicontroller.cpp"
 #include "util/math.h"
 
+MidiInputHandlerController::MidiInputHandlerController(
+        const LegacyMidiControllerMapping& mapping,
+        const MidiInputMapping& inputMapping)
+        : m_mapping(mapping), m_inputMapping(inputMapping) {
+}
+
+void MidiInputHandlerController::disconnect() {
+    this->m_mapping.removeInputMapping(this->m_inputMapping.key.key);
```

Comment:
Not a style we use in mixxx. it's already handled by the `m_` prefix.
```suggestion
    m_mapping.removeInputMapping(m_inputMapping.key.key);
```